build.gradle.kts

```diff
@@ -71,107 +73,208 @@ tasks.jar.configure {
+    }
+    create("appengine-appyaml") {
+      id = "$group.appengine-appyaml"
+      implementationClass =
+        "com.google.cloud.tools.gradle.appengine.appyaml.AppEngineAppYamlPlugin"
+      displayName = "App Engine Gradle Plugin"
+      description =
+        "This Gradle plugin provides tasks to build and deploy Google App Engine applications."
+    }
+    create("appengine-flexible") {
+      id = "$group.appengine-flexible"
+      implementationClass =
+        "com.google.cloud.tools.gradle.appengine.appyaml.AppEngineAppYamlPlugin"
+      displayName = "App Engine Gradle Plugin"
+      description =
+        "This Gradle plugin provides tasks to build and deploy Google App Engine applications."
+    }
+    create("appengine-standard") {
+      id = "$group.appengine-standard"
+      implementationClass =
+        "com.google.cloud.tools.gradle.appengine.standard.AppEngineStandardPlugin"
+      displayName = "App Engine Gradle Plugin"
+      description =
+        "This Gradle plugin provides tasks to build and deploy Google App Engine applications."
+    }
+    create("source-context") {
+      id = "$group.source-context"
+      implementationClass =
+        "com.google.cloud.tools.gradle.appengine.sourcecontext.SourceContextPlugin"
+      displayName = "App Engine Source Context Gradle Plugin"
+      description =
+        "This Gradle plugin injects source code context for cloud debugging."
+    }
+  }
+}
+// </editor-fold>
+
+// <editor-fold defaultstate="expanded" desc="Testing">
+tasks.test {
   testLogging {
     showStandardStreams = true
     exceptionFormat = FULL
   }
 }
 
-sourceSets {
-  create("integTest") {
-    compileClasspath += main.get().output
-    runtimeClasspath += main.get().output
-  }
+val integTestSourceSet = sourceSets.create("integTest") {
+  compileClasspath += sourceSets.main.get().output
+  runtimeClasspath += sourceSets.main.get().output
 }
 
 configurations {
-  named("integTestCompile").get().extendsFrom(testCompileClasspath.get())
-  named("integTestRuntime").get().extendsFrom(testRuntimeClasspath.get())
+  "integTestImplementation" {
+    extendsFrom(testImplementation.get())
+  }
+  "integTestRuntimeOnly" {
+    extendsFrom(testRuntimeOnly.get())
+  }
 }
 
-tasks.register<Test>("integTest") {
-  testClassesDirs = sourceSets.getByName("integTest").output.classesDirs
-  classpath = sourceSets.getByName("integTest").runtimeClasspath
+val integTest by tasks.registering(Test::class) {
+  testClassesDirs = integTestSourceSet.output.classesDirs
+  classpath = integTestSourceSet.runtimeClasspath
   outputs.upToDateWhen { false }
 }
-/* TESTING */
+// </editor-fold>
+
+// <editor-fold defaultstate="expanded" desc="Releasing">
+val generatePom by tasks.registering {
+  description =
+    "Generates all the pom files to ${buildDir.name}/publications/ for manual inspection"
+  group = "release"
+  dependsOn(tasks.withType<GenerateMavenPom>())
+}
```

Comment:
I think this is not necessary as one can just run `gradlew prepareRelease` and get an even better view, or browse the raw the maven repo as a result of `gradlew publish`.

Reply:
There was generatePom task befor this PR, so I keep it for backward compatibility

Reply:
No, there was a `writePom` task that is now replaced with `generatePomFileForPluginMavenPublication`.